ourjapanlife/findadoc-server
# feat: Add Firebase to the project

# What changed
1. Adds Firebase to the project
2. Updates the .env.sample fields with Firebase config variables
3. Updates the README with Firebase CLI info
4. Initializes the Firebase app in `index.ts`


# Testing instructions
1. Follow the steps in the README to ensure that you can log into Firebase through the CLI.
2. Confirm that your .env variables are working after copying the values from [Firebase Console](https://console.firebase.google.com/).

# TODO
Analytics and DB are defined in `index.ts` but not actually used yet.
